site-example.yml

```diff
@@ -4,10 +4,12 @@ Site: "Site Title"                                                      # Requir
 Output: "docs"                                                          # Required - Output directory where html files are generated
 Logo: "https://<domain.com>/url-to-logo.svg"                            # Site Branding
 SEO Author: "Author Name"                                               # Optional - Meta author tag for SEO
+Language: "English"                                                     # Required - Language of the docs
```

Comment:
Maybe it might be a good idea to indicate (either on the key name or the comment for this field) that this is the current language of the docs for the pages listed in this yml file.

Reply:
@kaankabalak will update the comment to convey it correctly, thanks